### PR TITLE
Feature/add markdown parser

### PR DESCRIPTION
**Pull request by kstarrmitche**
Done or close to it:
- [x] Autocompletion in the editor
- [x] highlighting in the editor (thanks to @perrywarner :) )
- [x] unit tests for the MarkdownService (though more are obviously better!)
- [x] MarkdownService as a whole 
- [x] crying 😸 

Need to do for future pull request
- [ ] fix scrolling on preview div so it doesn't overflow
- [ ] Add highlighting to preview div
- [ ] Add support for different language options within code fences
- [ ] still need to test if GFM works
- [ ] add A LOT more unit tests
- [ ] look out for any security holes when parsing html (Angular covers this for us, but it might be good to read more into it)

but even with the things done, I feel like this is a pretty good place for me to stop and take a rest...lmao.

# Current Tests
image

# Progress...
image

### CODE REVIEW

**Comment by kstarrmitche** [comment]
[Provide markdown-html conversions](https://trello.com/c/9uYD554J/37-provide-markdown-html-conversions)

File: README.md [hunk 1]
```diff
@@ -5,26 +5,19 @@ Everything in this project is WIP! including this readme!
 
 # How to get up and running
 1) Install `node` and `npm` through [this link](https://docs.npmjs.com/getting-started/installing-node).
-
```
Comment (perrywarner) [inline README.md:8]: Why did you delete this section of the README?

File: src/app/services/markdown-service.service.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@angular/core';
```
Comment (perrywarner) [inline src/app/services/markdown-service.service.ts:1]: Awesome work with the test spec :)

File: README.md (hunk 1 above)
Comment (kstarrmitche) [reply README.md:8]: These were just suggestions for ides/extensions. They're not really necessary for setting up our project :)

File: src/app/services/markdown-service.service.ts (hunk 2 above)
Comment (kstarrmitche) [reply src/app/services/markdown-service.service.ts:1]: Thanks dood :) I wanna add a few more but i think we're good for now

**Review by perrywarner — APPROVED** [review]
The list of 6 items on the todo list is all future tasks. I will add these tasks to Trello.

Great work on what was done, Kathy :)
